### PR TITLE
Refactor: simplify scripts

### PR DESCRIPTION
**Pull request by Borda**
# Before submitting

- [x] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you make sure to update the docs?
- [ ] Did you write any new necessary tests?

## What does this PR do?

working with bash/shell is just pain :smiley: 

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by Borda** [comment]
@ethanwharris cold ou pls check that the thumb pat is correct after refactor :rabbit:

**Review by awaelchli — APPROVED** [review]

File: .github/workflows/ci_code-format.yml [hunk 1]
```diff
@@ -10,19 +10,19 @@ on:  # Trigger the workflow on push or pull request, but only for the main branc
...
+#      - uses: actions/checkout@master
+#      - uses: actions/setup-python@v2
+#        with:
+#          python-version: 3.7
+#      - name: Install dependencies
+#        run: pip install flake8 && pip list
+#        shell: bash
+#      - name: PEP8
+#        run: |
+#          flake8 .
```
Comment (justusschock) [inline .github/workflows/ci_code-format.yml:25]: Can we remove this instead?

**Review by justusschock — COMMENTED** [review]
Can we not include the thumbnail as png here and maybe store it on s3? Once added we never get rid of it from the history and since it's binary every change will be tracked as a new file.

I know that this file is not that large but since we already have a quite large history and every bit counts, I'd rather not add this

**Comment by Borda** [comment]
> I know that this file is not that large but since we already have a quite large history and every bit counts, I'd rather not add this

I would not have much concern about thumbnails, but rather all other illustrations...
on the other hand, when someone accidentally deletes it in S3, it is lost... so that we allow images here but precommut preventing adding large files:
https://github.com/PyTorchLightning/lightning-tutorials/blob/5dd7b1c261a3fb6bd5788a9f13f7e95d3ede4987/.pre-commit-config.yaml#L21

**Review by justusschock — APPROVED** [review]

**Review by tchaton — APPROVED** [review]
LGTM !

**Review by ethanwharris — APPROVED** [review]
@Borda Seems to be working, LGTM :smiley:

**Pull request merged**